### PR TITLE
Write agenda plugin v0.1

### PR DESCRIPTION
**Pull request by JonanOribe**
A little script that allows the user to write key:value data in a CSV. This is the entry point for a Read_agenda plugin that recovers these data and future improves like a  reminder of events ,bearing in mind the dead line that they will have. For the next version Write_agenda will allow to add new lines to your agenda, at this moment every time you execute the script it will overwrite the old agenda with the new data

### CODE REVIEW

File: jarviscli/plugins/write_agenda.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from plugin import plugin
+import csv
+
+
+@plugin("write_agenda")
```
Comment (appi147) [inline jarviscli/plugins/write_agenda.py]:
```suggestion
@plugin("write agenda")
```

Comment (JonanOribe) [reply jarviscli/plugins/write_agenda.py]: Really?? I hate Travis jajaj.Thanks for the help!!

**Comment by JonanOribe** [comment]
I think that is something about the way I make the "While" at line 11

**Comment by appi147** [comment]
Add read agenda in this file, from PR #451

**Comment by appi147** [comment]
Yes that is a problem, but we follow this naming

**Pull request merged**